hoomd/md/many_body.py

```diff
@@ -0,0 +1,343 @@
+        nlist (:py:mod:`hoomd.md.nlist`): Neighbor list
+        r_cut (float): Default cutoff radius (in distance units).
+        r_on (float): Default turn-on radius (in distance units).
+        mode (str): Energy shifting mode.
+
+    :py:class:`RevCross` specifies that the revcross three-body potential
+    should be applied to every non-bonded particle pair in the simulation.
+    Despite the fact that the revcross potential accounts for the effects of
+    third bodies, it is actually just a combination of two body potential terms.
+    It can thus use type-pair parameters similar to those of the pair potentials.
+
+    The RevCross potential has been described in detail in
+    `S. Ciarella and W.G. Ellenbroek 2019 <https://arxiv.org/abs/1912.08569>`_.
+    It is based on a generalized-Lennard-Jones pairwise attraction to form bonds
+    between interacting particless:
+
+    .. math::
+        :nowrap:
+
+        \\begin{eqnarray*}
+        V_{ij}(r)  =  4 \\varepsilon \\left[ \\left( \\dfrac{ \\sigma}{r_{ij}} \\right)^{2n}
+        - \\left( \\dfrac{ \\sigma}{r_{ij}} \\right)^{n} \\right] \\qquad r<r_{cut}
+        \\end{eqnarray*}
+
+    with the following coefficients:
+
+    - :math:`\\varepsilon` - *epsilon* (in energy units)
+    - :math:`\\sigma` - *sigma* (in distance units)
+    - :math:`n` - *n* (unitless)
+    - :math:`m` - *m* (unitless)
+    - :math:`r_{\\mathrm{cut}}` - *r_cut* (in distance units)
+      - *optional*: defaults to the global r_cut specified in the pair command
+
+    Then an additional three-body repulsion is evaluated to compensate the bond
+    energies imposing single bond per particle condition:
+
+    .. math::
+        :nowrap:
+
+            \\begin{eqnarray*}
+            v^{\\left( 3b \\right)}_{ijk}=\\lambda \\epsilon\\,\\hat{v}^{ \\left( 2b \\right)}_{ij}
+            \\left(\\vec{r}_{ij}\\right) \\cdot \\hat{v}^{ \\left( 2b \\right)}_{ik}
+            \\left(\\vec{r}_{ik}\\right)~,\\
+            \\end{eqnarray*}
+
+    where the two body potential is rewritten as:
+
+    .. math::
+        :nowrap:
+
+            \\begin{eqnarray*}
+            \\hat{v}^{ \\left( 2b \\right)}_{ij}\\left(\\vec{r}_{ij}\\right) =
+            \\begin{cases}
+            1 & r \\le r_{min} \\\\
+            - \\dfrac{v_{ij}\\left(\\vec{r}_{ij}\\right)}{\\epsilon} & r > r_{min} \\\\
+            \\end{cases}
+            \\end{eqnarray*}
+
+    .. attention::
+
+        The RevCross potential models an asymmetric interaction between two
+        different chemical moieties that can form a reversible bond. This
+        requires the definition of (at least) two different types of particles.
+        A reversible bond is only possible between two different species,
+        otherwise :math:`v^{\\left( 3b \\right)}_{ijk}`, would prevent any bond.
+        In our example we then set the interactions for types A and B with
+        ``rev_c.params[[('A','B'),('A','B')]] = dict(sigma=0.0,n=0,epsilon=0,lambda3=0)``
```

Comment:
Unfortunately this renders poorly when my browser is set to half a screen. I am not sure of the best way to change this though.

Reply:
We could break the code block into two at the `=`.